src/errors/Messages.js

```diff
@@ -145,7 +145,6 @@ const Messages = {
 
   INTERACTION_ALREADY_REPLIED: 'The reply to this interaction has already been sent or deferred.',
   INTERACTION_NOT_REPLIED: 'The reply to this interaction has not been sent or deferred.',
-  INTERACTION_EPHEMERAL_REPLIED: 'Ephemeral responses cannot be deleted.',
```

Comment:
It should not be removed, it should be deprecated. Removing it would be a breaking change